### DISCUSSION TITLE
Releasing v10.2 in June 2022

### DISCUSSION THREAD

**Discussion by vhelin** [vhelin/wla-dx#507] [General]
I feel like we should wrap things up and release v10.2 in June 2022. If you guys have more bug reports, please let us know them asap.

For v10.3 I think I'll look into issue #7 (function style defines, https://github.com/vhelin/wla-dx/issues/7) and try to implement that. And perhaps, if it's doable given the current limitations in a reasonable amount of time, add support for 68000 (https://github.com/vhelin/wla-dx/issues/170).

Any objections / quick improvement ideas / comments?

**Comment by Stewmath** [comment]
I tried compiling my monster gameboy project with the latest sources and all was good, except that the ``.rombanks`` directive doesn't write the corresponding value to the cartridge header anymore. Apparently this was intentional so I guess there was a good reason for it. It's not ideal to have this kind of breaking change, but at least bgb warns about it so it's not like people won't notice. I would recommend adding the .romsize directive to the tests though.

Anyway, the gist is that I didn't encounter any actual bugs so it looks to be in good shape :+1:

**Comment by vhelin** [comment]
Thanks for testing the latest sources!

The reasoning behind that .ROMBANKS change can be found here: https://github.com/vhelin/wla-dx/issues/498

And that's why we have the CHANGELOG file that people can read what's changed and take action if something old has been changed. :)

```
            [GB ] WARNING: .ROMBANKS / .ROMBANKMAP don't write the ROM
                      size to $0148 any more, use .ROMSIZE for that.
```
